specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -2443,6 +2620,262 @@
           "description": "A textual explanation that can be used for display purposes to summarize the index value and category. For example, when the index value for Flight Delays is very good, the description will be \"Conditions are excellent for flying!\"."
         }
       }
+    },
+    "DailyHistoricalActualsResponse": {
+      "type": "object",
+      "properties": {
+        "historicalActuals": {
+          "type": "array",
+          "description": "Historical actuals for each requested day.",
+          "items": {
+            "$ref": "#/definitions/DailyHistoricalActuals"
+          }
+        }
+      }
+    },
+    "DailyHistoricalActuals": {
+      "type": "object",
+      "properties": {
+        "date": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Date and time of the current observation displayed in ISO 8601 format, for example, 2019-10-27T19:39:57-08:00."
+        },
+        "temperature": {
+          "$ref": "#/definitions/WeatherUnitMaxMinAvg",
+          "description": "Temperature values."
+        },
+        "degreeDaySummary": {
+          "$ref": "#/definitions/DegreeDaySummary",
+          "description": "Summary of heating or cooling degree day information"
+        },
+        "precipitation": {
+          "$ref": "#/definitions/WeatherUnit",
+          "description": "The amount of precipitation (liquid equivalent) that has fallen."
+        },
+        "snow": {
```

Comment:
The API response type ‘Snow’ should actually be ‘Snowfall’ just like AccuWeather’s Climo API has. Please update the API response accordingly and also the documentation (and sample response in documentation).